frontends/concrete-python/tests/execution/test_tfhers.py

```diff
@@ -0,0 +1,195 @@
+"""
+Tests execution of tfhers conversion operations.
+"""
+
+import pytest
+
+from concrete import fhe
+from concrete.fhe import tfhers
```

Comment:
I think it'd be better if we can do:
```
from concrete import fhe, tfhers
```

Reply:
This means a different module, and I'm not sure it makes sense as they would depend on each other